.gitignore

```diff
@@ -8,6 +8,7 @@
 src/clr-angular/.ng_build
 src/clr-angular/.ng_pkg_build
 /tests/tslint/dist
+src/clr-angular/package.json
```

Comment:
This file is getting deleted. Is there a reason this needs to be here?

Reply:
It was removed because its automatically generated during the build. Sometimes it was getting committed after being updated during releases, sometimes not. Point is, we don't need it except for build time and its generated. ng-packagr needs to have a version of the file for its build process.